sphinxcontrib/wavedrom_render_image.py

```diff
@@ -38,8 +38,12 @@ def render_wavedrom_py(node, outpath, bname, image_format):
     # Try to convert node, raise error with code on failure
     try:
         svgout = render(node["code"])
-    except JSONDecodeError as exception:
-        raise SphinxError("Cannot render the following json code: \n{} \n\nError: {}".format(node['code'], exception))
+    except Exception:
```

Comment:
Any specific reason why you didn't just go with `except:` instead? I don't mind, just curious if there's a difference.

I'm not a big fan of this catch-all exception, but I can see why you don't want to try out all the ways the rendering can fail and catch those exceptions here.

@wallento would it be possible to add some more elegant Exception handling/raising in the render method? Or do you have other ideas on how we should handle error conditions here?

Reply:
`except:` will also catch very low level stuff like InterruptedException (ctrl-c), which is not what we want here. We only want "normal" exceptions.

There are some subtle "Syntax errors" in the permissive JSON supported by sphinx-wavedrom that I want to catch and produce decent error messages for.

Consider the below, both parse correctly in JSON, but the result is different. Notice the space after "node:".

```json
{
   node:"bar"
}
```

vs:

```json
{
   node: "bar"
}
```

Reply:
> except: will also catch very low level stuff like InterruptedException (ctrl-c), which is not what we want here. We only want "normal" exceptions.

Thanks, I've learned something today :)

Reply:
@bavovanachte Kind reminder: anything else to fix in this PR? Comments?

Reply:
@bavovanachte Ping?